### ISSUE TITLE
Onnx Convert Error：MNN not support Clip 

### ISSUE THREAD

**Issue by chrisYooh** [alibaba/MNN#136]
**Questions**
Hi, I want to know how can I convert onnx model with【Clip】to MNN when MNN not support 【Clip】?
1. Will you add 【Clip】operation？
2. Or should I modify MNNConvert source code？

**Detail Infomation**
My step：pytorch --> onnx --> mnn
1 Pytorch has operation 【Relu6】
2 And when convert pytorch to onnx，【Relu6】 changed to 【Clip】
（This is just OK，the operation is the same）
3 But when I convert onnx to mnn，the terminal tell me mnn doesn't support 【Clip】

**Error Logs**

`Start to Convert Other Model Format To MNN Model...`

`[16:14:24] [MNN forder]/tools/converter/source/onnx/onnxConverter.cpp:29: ONNX Model ir version: 4`

`libc++abi.dylib: terminating with uncaught exception of type Error: [16:14:24] [MNN forder]/tools/converter/source/onnx/onnxConverter.cpp:83: Check failed: opConverter ==> MNN Converter NOT_SUPPORTED_OP: [ Clip ]`

`[My convert sh forder]/sh_onnx2mnn.sh: line 10: 77671 Abort trap: 6           ${MNN_CONVERTER} -f ONNX --modelFile ${ONNX_FILE_PATH} --MNNModel ${MNN_FILE_PATH} --bizCode MNN
`

**Comment by li-qing** [comment]
Clip is  supported now, check out latest commit plz.

**Comment by chrisYooh** [comment]
**Convert Success**
I check out the latest commit and try again. The convert passed.

**Inference Net Create Session Error**
But I can't ensure whether the convert result is right.
I tried two of our models, when run inference, it sopped at an assert.

<img width="708" alt="image" src="https://user-images.githubusercontent.com/16423747/60638643-bce25580-9e51-11e9-847f-e7b34883974f.png">

<img width="509" alt="image" src="https://user-images.githubusercontent.com/16423747/60638812-704b4a00-9e52-11e9-8172-85ec65446f26.png">

`Error for .../MNN/source/shape/ShapeInstanceNorm.cpp, 18
Assertion failed: (res), function onComputeSize, file ../MNN/source/shape/ShapeInstanceNorm.cpp, line 18.`

**Doing**
I know the information may be not enouth.
I'll try to find more detail informations.
If you have some suggest, please tell me. Thanks~

**Comment by jxt1234** [comment]
tools/converter/source/onnx/onnxConverter.cpp
static std::set<std::string> treatInitializerOp{"Conv", "Upsample", "Reshape", "Const", "Gemm"}
->
static std::set<std::string> treatInitializerOp{"Conv", "Upsample", "Reshape", "Const", "Gemm", "BatchNormalization"};

**Comment by chrisYooh** [comment]
Great！
The convert successed and the inference right!

**Comment by PTL2011** [comment]
hello. could you tell me how you change relu6 to clip in pytorch?
